samples/hello/main.c

```diff
@@ -5,7 +5,7 @@
 #include <windows.h>
 #include "stdio.h"
 
-void main(void)
+int main()
```

Comment:
As discussed on Discord, all of these have to be either:
```c
int main(void)
```
or
```c
int main(int argc, char* argv[])
```

see https://www.pdf-archive.com/2014/10/02/ansi-iso-9899-1990-1/ansi-iso-9899-1990-1.pdf page 6 (Page 16 in PDF).

Both styles have their pros and cons. I'd suggest using `(void)` for now.
If we ever introduce command line argument parsing from debuggers, we can add new samples which showcase the `argc`/`argv` style.

Reply:
Changed to (void)